src/main/java/website/frontrow/game/Game.java

```diff
@@ -214,6 +237,51 @@ public void levelWon()
     @Override
     public void levelLost()
     {
+    	stop();
+    	if(levelPack.size() != 1)
+    	{
+    		levelPack.set(currentIndex+1,gameOver);
+    		gameOver();
+    	}
+    }
+    
+    /**
+     * Returns whether or not all players are dead.
+     * @return b boolean
+     */
+    public boolean playersDead()
+    {
+    	if(!this.isRunning())
+    	{
+    		return false;
+    	}
+    	for(Player p : this.players)
+    	{
+    		if (p.hasLives())
+    		{
+    			return false;
+    		}
+    	}
+    	return true;
+    }
+    
+    /**
+     * Adds Game Over message to log.
+     */
+    public void gameOver() 
+    {
+    	currentLevel = gameOver;
```

Comment:
I had this once before as well, it's probably the difference between tabs and spaces. See if you can't give all lines the same type of indentation.